.pre-commit-config.yaml

```diff
@@ -1,6 +1,18 @@
 ---
-exclude: ^(\.[^/]*cache(__)?/.*|\.coverage)$
+exclude: ^(\.[^/]*cache(__)?/.*|(.*/)?\.coverage)$
 repos:
+  - repo: https://github.com/executablebooks/mdformat
```

Comment:
To answer an earlier question you asked: I'm OK to keep that hook as part of this PR